spec/classes/repo_spec.rb

```diff
@@ -13,14 +13,18 @@
           .that_notifies('Anchor[pulpcore::repo]')
       end
 
-      it 'configures the pulpcore module' do
-        is_expected.to contain_package('pulpcore-dnf-module')
-          .with_name('pulpcore')
-          .with_ensure(/^el\d+/)
-          .with_enable_only(true)
-          .with_provider('dnfmodule')
-          .that_requires('File[/etc/yum.repos.d/pulpcore.repo]')
-          .that_notifies('Anchor[pulpcore::repo]')
+      if os_facts[:op]['release']['major'] == '8'
```

Comment:
```suggestion
      if os_facts[:os]['release']['major'] == '8'
```

;)

Reply:
I'll just leave this here until the other parts have been sorted.